thumbnail-service/src/main/java/no/sikt/nva/thumbnail/util/ResizeImage.java

```diff
@@ -0,0 +1,134 @@
+package no.sikt.nva.thumbnail.util;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.RenderingHints;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import javax.imageio.ImageIO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ResizeImage {
```

Comment:
ImageResizer a better name for the class? Also here, I would have preferred one public method, resize() with options if needed, and avoid store state as instance variable and rather pass them between the private methods.

Reply:
fixed